src/box/txn_limbo.c

```diff
@@ -1086,6 +1091,84 @@ txn_limbo_filter_request(struct txn_limbo *limbo,
 	}
 }
 
+static uint32_t sync_system_space_ids[] = {
+	/* BOX_VINYL_DEFERRED_DELETE_ID, local space*/
+	BOX_SCHEMA_ID,
+	BOX_COLLATION_ID,
+	BOX_VCOLLATION_ID,
+	BOX_SPACE_ID,
+	BOX_VSPACE_ID,
+	BOX_SEQUENCE_ID,
+	/* BOX_SEQUENCE_DATA_ID, synchronized by synchronous space operations */
```

Comment:
Please, mention these two exceptions in the docbot request with a small comment on why they are kept async.

Reply:
Fixed.